src/eei.cpp

```diff
@@ -722,8 +755,18 @@ inline int64_t maxCallGas(int64_t gas) {
         break;
       }
 
+#if HERA_DEBUGGING
+      string methodName;
+      switch (kind) {
+      case EEICallKind::Call: methodName = "call"; break;
+      case EEICallKind::CallCode: methodName = "callCode"; break;
+      case EEICallKind::CallDelegate: methodName = "callDelegate"; break;
+      case EEICallKind::CallStatic: methodName = "callStatic"; break;
+      }
+#endif
+
       HERA_DEBUG <<
-        import->base << " " << hex <<
+        methodName << " " << hex <<
```

Comment:
this needs to be conditionally compiled with HERA_DEBUG for release builds to work correctly.

Reply:
That's fine, the main question in this PR is the naming and general idea.